src/go/k8s/pkg/console/store.go

```diff
@@ -188,6 +219,10 @@ func (s *Store) getKafkaNodeCertKey(cluster *redpandav1alpha1.Cluster) string {
 	return fmt.Sprintf("%s-%s-%s", cluster.GetNamespace(), cluster.GetName(), "kafka-node")
 }
 
+func (s *Store) getAdminAPINodeCertKey(cluster *redpandav1alpha1.Cluster) string {
+	return fmt.Sprintf("%s-%s-%s", cluster.GetNamespace(), cluster.GetName(), adminAPINodeCertSuffix)
+}
+
```

Comment:
NIT: This function should be introduced in previous commit.

Reply:
Updated